docs/Common/Enums/index.md

```diff
@@ -1305,10 +1308,25 @@
 |CLOUD|1||
 |BOTH|2||
 
+<<<<<<< HEAD
+### TransmissionType
+
+|Name|Value|Description|
+|:---|:----|:----------|
+|MANUAL|0|Manual transmission.|
+|AUTOMATIC|1|Automatic transmission.|
+|SEMI_AUTOMATIC|2|Semi automatic transmission.|
+|DUAL_CLUTCH|3|Dual clutch transmission.|
+|CONTINUOUSLY_VARIABLE|4|Continuously variable transmission(CVT).|
+|INFINITELY_VARIABLE|5|Infinitely variable transmission.|
+|ELECTRIC_VARIABLE|6|Electric variable transmission.|  
+|DIRECT_DRIVE|7|Direct drive between engine and wheels.|
+=======
 ### CapacityUnit
```

Comment:
```suggestion

### CapacityUnit
```

Reply:
@ShobhitAd fixed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/214/commits/d20c6eb18a1a17d0b0da563deb346b018ff6fdd5